src/profilers.cpp

```diff
@@ -230,6 +224,9 @@ namespace aperf {
             temp_data = match.suffix().str();
           
           }
+
+          close(pipe_fd[0]);
```

Comment:
It's better than previously, but I'd still move it closer to the end of the ```while((bytes_read...))``` loop, e.g. immediately before ```std::string parsed_data;``` :)

Reply:
:) Sorry about that, I didn't pay attention and just put it at the end of the closest while loop. Done :+1: